less/index.less

```diff
@@ -52,8 +52,176 @@ table {
 * {
     box-sizing: border-box;
 }
-
+@font:1.6rem;
 html, body {
     height: 100%;
+    font-size: 62.5%;
+    
+}
+.flexbox-centered(){
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.flexbox-end(){
+    display: flex;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+}
+.flexbox-between(){
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+.Container{
+    max-width: 900px;
+    width: 100%;
+    margin: 0 auto;
+    
+    @media(max-width:500px){
+        width:90%;
+        margin: 0 auto;
+        
+    }
+}
+.image{
+    
+}
+nav{
+    .flexbox-end;
+    margin-bottom: 30px;
+    margin-top: 30px;
+    @media(max-width:500px){
+    justify-content: center;
+    }
+    
+a {
+     margin-left:2%; 
+     font-size: @font;
+     text-decoration: none;
+    color: gray;
+    }
+    
+}
+.intro{
+    background-color:#E9ECEF;
+    padding: 5% 3% 7% 3%;
+    margin-bottom: 20px;
+    h1{
+        font-size: 6rem;
+
+    }
+    p{
+        margin-top: 15px; 
+        font-size: @font;
+    }
+    button{
+        margin-top: 10px;
+        padding: 5px 10px; 
+        background-color: #808080;
+        color: white;
+        
+    }
+}
+.listen-Good{
+    
+    .flexbox-between;
+    margin-bottom: 20px; 
+    
+    @media(max-width:500px){
+        
+    flex-direction: column; 
+    }
+    .listen{
+        width: 50%;
+        h2{
+            font-size: 3rem;
+            margin-bottom: 10px;
+        
+        }
+        p{
+            font-size:1.5rem;
+        }
+        @media(max-width:500px){
+        width: 100%;
```

Comment:
Look into a VS Code extension that will tidy up the formatting on save. Line 146 needs to be indented, and an extension will help with that.